packages/debian/cloud-init.postinst

```diff
@@ -1,16 +0,0 @@
-#!/bin/sh
-cleanup_lp1552999() {
-    local oldver="$1" last_bad_ver="0.7.7~bzr1178"
```

Comment:
Given the release version gate is >8 years ago, this makes sense to drop. There are many interim releases/upgrade that should have taken care of this upgrade path.